rust/agents/relayer/src/settings/mod.rs

```diff
@@ -157,7 +158,9 @@ decl_settings!(Relayer,
     Raw {
         /// Database path (path on the fs)
         db: Option<String>,
-        // The name of the origin chain
+        // Comma separated list of origin chains.
+        originchainnames: Option<String>,
```

Comment:
What is the advantage of allowing origin and destination chain names to be specified separately?

Reply:
Sorry, how is it backwards compatible? Don't you need to specify origin chain names now?

> the future possibility of having central relayers for certain destination chains
> It also allows people to split up their instances if that is important to them for some reason.

These feel possible via blacklist, no?

IMO separate origin/destination chain config feels more complicated for the typical use case

Reply:
From our discussion this morning at standup:

Instead of `originchainnames` and `destinationchainnames` we should accept just a single list of chains (but maintain support for the old `originchainname` and `destinationchainnames` config for now).

If an operator has a new chain A and wants to connect it with B and C, the most likely case is they want to relay A <-> B and A <-> C, but NOT B <-> C and that can be easily handled with a blacklist. Since this does not change the fact that all of the chains still need to be indexed, it would not need any additional code changes for efficiency.